app/scripts/directives/mobileServiceInstanceRow.js

```diff
@@ -49,7 +49,7 @@
         row.mobileClient = mobileClient;
 
         row.bindingMeta = {
-          generateName: _.get(row, 'mobileClient.metadata.name').toLowerCase() + '-' + _.get(row, 'serviceClass.spec.externalMetadata.displayName').toLowerCase() + '-',
+          generateName: _.get(row, 'mobileClient.metadata.name').toLowerCase().replace(/ /g, "-") + '-' + _.get(row, 'serviceClass.spec.externalMetadata.displayName').toLowerCase().replace(/ /g, "-") + '-',
```

Comment:
@JameelB has a different fix in her mobile row changes -> https://github.com/aerogear/origin-web-console/commit/b9f681c2d25c0e39e91d247801617af9fbe5c3be#diff-919cca39c4330b7066c527662c1e3d7cR52
I think it's a simpler fix wdyt?

Reply:
yeah, prefer that.